src/chapters/setup-intro-data-types/exercises.rst

```diff
@@ -1,2 +1,152 @@
+#. From the URL dropdown options, select your fork of ``java-web-dev-exercises``, 
+   along with an appropriate source destination directory (i.e. a folder where you’ve stored other projects for this class).
+#. When asked “Would you like to create an IDEA project…” select *Yes*, 
+   and then accept all of the default options that are presented.
+
+With that, you’re ready to go!
+
+Here’s a video walking through the last 4 steps above:
+
+.. raw:: html
+
+   <div class="youtube-wrapper">
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/OPCaYVXRm_c" frameborder="0" allowfullscreen>
+
+.. raw:: html
+
+   </iframe>
+
+.. raw:: html
+
+   </div>
+
+Troubleshooting
+---------------
+
+ClassNotFoundException
+^^^^^^^^^^^^^^^^^^^^^^
+
+If you experience ``java.lang.ClassNotFoundException`` when trying to
+run code after setting up the project, follow these steps: 
+
+#. Select *File > Close Project*.
+#. From the IntelliJ modal window, click the *X* next to ``java-web-dev-exercises`` in the left-hand pane.
+#. From the same modal window, select *Import Project* from the right-hand pane.
+#. Follow the steps that IntelliJ guides you through, accepting all defaults. When prompted to overwrite IntelliJ settings files, you confirm that you want to do so.
+
+
+Instructions
+------------
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project. Create a
+new class for each numbered exercise. You may name the classes whatever you like, but use
+proper :ref:`naming-conventions` and make sure that the file name matches the class name.
+
+Creating a Package and Classes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here is how to create a new package to store these exercises, and how to
+create new classes within this package:
+
+#. Click on the folder ``src`` in the Project pane, then right click and
+   select *New* and then select *Package*.
+
+   .. figure:: figures/newPackage.png
+      :alt: New Package
+
+      New Package
+
+#. Name the package “exercises”.
+
+   .. figure:: figures/namePackage.png
+      :alt: Name Package
+
+      Name Package
+
+#. Then click on the newly-created folder ``exercises`` and right click
+   to select *New* and then select *Class*.
+
+   .. figure:: figures/newClass.png
+      :alt: New Class
+
+      New Class
+
+#. Name this what you will name your class (for example, in the
+   4th exercise below, you might name the class ``Alice``).
+
+   .. figure:: figures/nameClass.png
+      :alt: Name Class
+
+      Name Class
+
+
+   .. note:: 
+      You will be asked whether you want to add this file to Git.
+      Press the “Yes” button.
+   
+   .. figure:: figures/addToGit.png
+      :alt: Add class to Git
+
+      Add class to Git
+
+5. The new class is now created! You can proceed to write code within
+   it. (Don’t forget to write the ``main`` method!)
+
+   .. figure:: figures/newClassCreated.png
+      :alt: Ready to start
+
+      Ready to start
+
+Exercises
```

Comment:
Since these are the first exercises in Java, I'd recommend providing some heavy guidance for exercise 1 and less specific but still relatively complete guidance for exercise 2.  Exercises 3 & 4 are fine as-is, but I'd add a reference back to the Strings method page for exercise 4 "if you need a quick review of the Java String methods."

Reply:
good call. will change

Reply:
I like the updates.  The exercises are much more approachable now.